packages/terra-tabs/src/clinical-lowlight-theme/Tabs.module.scss

```diff
@@ -1,5 +1,6 @@
 :local {
   .clinical-lowlight-theme {
+    --terra-tabs-remove-button-color: #006fc3;
```

Comment:
you need to this variable in fusion theme as well to match the color of fusion components.

Reply:
Done https://github.com/cerner/terra-framework/commit/fcb1925802030dfb5be6bf821a0a978a5c82d40c